src/main/java/org/opentripplanner/middleware/controllers/api/AbstractUserController.java

```diff
@@ -0,0 +1,108 @@
+package org.opentripplanner.middleware.controllers.api;
+
+import com.auth0.exception.Auth0Exception;
+import com.beerboy.ss.ApiEndpoint;
+import org.eclipse.jetty.http.HttpStatus;
+import org.opentripplanner.middleware.auth.Auth0Connection;
+import org.opentripplanner.middleware.auth.Auth0UserProfile;
+import org.opentripplanner.middleware.models.AbstractUser;
+import org.opentripplanner.middleware.persistence.TypedPersistence;
+import org.opentripplanner.middleware.utils.JsonUtils;
+import spark.Request;
+import spark.Response;
+
+import static com.beerboy.ss.descriptor.MethodDescriptor.path;
+import static org.opentripplanner.middleware.auth.Auth0Users.*;
```

Comment:
I think we generally try to avoid wildcard imports.

Reply:
Fixed in c424e4a.